### DISCUSSION TITLE
GEN3: Pokémon reported as illegal when viewing the PC, but when you click on the checkmark, it becomes legal...?

### DISCUSSION THREAD

**Discussion by Lorenzooone** [kwsch/PKHeX#3828] [Q&A] [answered]
**Describe the problem**
I have three Pokémon on my save file which display the illegal symbol when you look at them using the PC view. If you click on the exclamation mark, they suddenly become legal...? 02 27 2023 build of PkHex on Windows.

**To Reproduce**
[POKEMONL_WEIRD_LEGAL.zip](https://github.com/kwsch/PKHeX/files/10891782/POKEMONL_WEIRD_LEGAL.zip)

**Expected behavior**
Either flag them as illegal, and tell me why they are so, or do not mark them as illegal...? It's... confusing.

**Screenshots**
![one_legal](https://user-images.githubusercontent.com/7537905/222959875-f4f1b5e2-8f47-4633-8f13-8a4ac5cac936.PNG)
![two_legal](https://user-images.githubusercontent.com/7537905/222959873-2fa9bf51-9ef7-4ccc-983d-3931e027c43c.PNG)

**Additional context**
I put them in the PC in-game from when they were on my team...? I haven't done anything exceptional with them...?

**Comment by Lusamine** [comment, marked as answer]
Not a bug.  Control + right click the Pokémon to bring up the Legality option.
![image](https://user-images.githubusercontent.com/30205550/222972592-76a2341b-7ad4-4b7c-942e-1d7dd7c63230.png)

Your Pokémon has Pokerus strain 0 which is being corrected when it's loaded into the active pane to the left.

**Comment by kwsch** [comment]
The PKM Editor fixes some mismatching issues when you view it in the editor.

Hold control when right clicking the box slot and you can check legality without loading to tabs.
